### ISSUE TITLE
Unable receive udp/coap message

### ISSUE THREAD

**Issue by donaldkohcw** [openthread/openthread#11322]
Hello, 

I am having an issue receiving any inbound messages on my OTBR interfaced with an ESP OT-RCP. I have no issues sending messages to Thread devices.

I also have no issues using ping between them. Is there any guidance on how to enable inbound message reception on obtr? Thanks.

Regards,
dk

This is my Linux ipaddr:

sudo ot-ctl ipaddr
fd4a:2c7:139b:b96:0:ff:fe00:fc11
fd76:f765:9e44:1:157f:4e49:f534:fd13
fd4a:2c7:139b:b96:0:ff:fe00:fc10
fd4a:2c7:139b:b96:0:ff:fe00:fc38
fd4a:2c7:139b:b96:0:ff:fe00:fc00
fd4a:2c7:139b:b96:0:ff:fe00:a000
fd4a:2c7:139b:b96:728:f37c:4e8a:b25
fe80:0:0:0:7067:2618:b927:3cce
Done

Ping example from device to otbr sucessfully:

 ping fd4a:2c7:139b:b96:0:ff:fe00:fc11

16 bytes from fd4a:2c7:139b:b96:0:ff:fe00:fc11: icmp_seq=3 hlim=64 time=79ms
1 packets transmitted, 1 packets received. Packet loss = 0.0%. Round-trip min/avg/max = 79/79.0/79 ms.
Done
